cypress/integration/LockedSelector/index.js

```diff
@@ -300,6 +300,7 @@ And('you see the enrollment page', () => {
 And('you reset the enrollment selection', () => {
     cy.get('[data-test="reset-selection-button"]')
         .should('have.length.greaterThan', 2);
+    cy.wait(500);
```

Comment:
Hi @simonadomnisoru, do you think it make sense to wait for an element instead of a fixed timeout? 
For example we can wait for the content `dhis2-capture-enrollment-page-content` is visible before clicking the reset button

Reply:
Hey @huocha , 

That is a good question, thank you for the feedback!
 If you want to wait for a specific element to appear in the DOM you have to specify a fixed `timeout` according to [Cypress docs ](https://docs.cypress.io/faq/questions/using-cypress-faq#Some-of-my-elements-animate-in-how-do-I-work-around-that). In this case it will be something like this 
`cy.get('[data-test="enrollment-page-content"]', { timeout: 500 }).should('be.visible') `. 

I don't have a preference with which solution to choose. I will let the rest of the team decide. What do you think @JoakimSM?

Reply:
Hey, 

This is a good suggestion @JoakimSM. 
Looking for the text `Enrollment Dashboard` is not enough and the Scenarios are still flaky.
I am looking for the text `Person Profile` which is displayed in the Profile widget. With this check, I am sure that the page has finished loading and the tests are passing. I run the Cypress test 5 times and they are all green [branch Cypress Dashboard ](https://dashboard.cypress.io/projects/322xnh/runs?branches=%5B%7B%22label%22%3A%22DHIS2-11067%22%2C%22suggested%22%3Afalse%2C%22value%22%3A%22DHIS2-11067%22%7D%5D&committers=%5B%5D&flaky=%5B%5D&page=1&status=%5B%5D&tags=%5B%5D&timeRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D)
Thank you!